lib/location/method_cloud_location.c

```diff
@@ -118,12 +116,11 @@ static void method_cloud_location_positioning_work_fn(struct k_work *work)
 #else
 	struct location_data location;
 	struct location_data location_result = { 0 };
-	int64_t scan_time;
 	struct cloud_service_pos_req params = {
 		.cell_data = scan_cellular_info,
 		.wifi_data = scan_wifi_info,
 		.service = (cell_config != NULL) ? cell_config->service : wifi_config->service,
-		.timeout_ms = used_timeout_ms
+		.timeout_ms = SYS_FOREVER_MS
```

Comment:
I think we need something other than infinite here to avoid getting stuck if data transfer has issues.
Opinions?

Reply:
As discussed, I'll check if we could set here the remaining time from global location request timeout.

Reply:
Timeout handling modified